toolkit/tools/pkg/imagecustomizerlib/customizepackages.go

```diff
@@ -82,6 +82,11 @@ func addRemoveAndUpdatePackages(buildDir string, baseConfigPath string, config *
 		}
 	}
 
+	err = cleanTdnfCache(imageChroot)
```

Comment:
Place this code under a `if needRpmsSources {` check. Otherwise, it will always run.

Reply:
Done.